docs/CHANGELOG.asciidoc

```diff
@@ -77,6 +78,353 @@ Review important information about the {kib} 8.x releases.
+* Fixes an issue where label truncation in heat map legends was not working properly in *Lens* ({kibana-pull}195928[#195928]).
+* Fixes an issue where the color picker and axis side settings were incorrectly available in the breakdown dimension editor for XY charts in *Lens* ({kibana-pull}195845[#195845]).
+* Fixes the tooltip position on faceted charts in *Vega* ({kibana-pull}194620[#194620]).
+* Fixes the filter out legend action for ES|QL visualizations ({kibana-pull}194374[#194374]).
+* Fixes element sizing issues in full screen mode in *Vega* ({kibana-pull}194330[#194330]).
+* Fixes the default cell text alignment setting for non-numeric field types in *Lens* ({kibana-pull}193886[#193886]).
+* Limits the height of the query bar input for long KQL queries ({kibana-pull}193737[#193737]).
+* Makes the title correctly align left after removing an icon in **Lens** metric charts ({kibana-pull}191057[#191057]).
+* Fixes a "No data" error caused by the "Collapse by" setting in **Lens** metric charts ({kibana-pull}190966[#190966]).
+* Fixes an issue causing the color of a cell to disappear when clicking the "Expand cell" icon in *Lens* ({kibana-pull}190618[#190618]).
+* Removes unnecessary index pattern references from Lens charts ({kibana-pull}190296[#190296]).
+* Fixes several accessibility issues ({kibana-pull}188624[#188624]).
+Data ingestion and Fleet::
+* Revert "Fix client-side validation for agent policy timeout fields" ({kibana-pull}194338[#194338]).
+* Adds proxy arguments to install snippets ({kibana-pull}193922[#193922]).
+* Rollover if dimension mappings changed in dynamic templates ({kibana-pull}192098[#192098]).
+Discover::
+* Fixes an issue with search highlighting ({kibana-pull}197607[#197607]).
+* Correctly pass embeddable filters to the Surrounding Documents page ({kibana-pull}197190[#197190]).
+* Fixes trailing decimals dropped from client side validation messages ({kibana-pull}196570[#196570]).
+* Fixes several validation issues and creates an expression type evaluator for ES|QL queries ({kibana-pull}195989[#195989]).
+* Fixes duplicate autocomplete suggestions for `WHERE` clauses and suggestions with no space in between in ES|QL queries ({kibana-pull}195771[#195771]).
+* Improves variable and field name handling in ES|QL queries ({kibana-pull}195149[#195149]).
+* Fixes an issue where the Unified Field List popover could get cut off ({kibana-pull}195147[#195147]).
+* Fixes the width for saved object type columns ({kibana-pull}194388[#194388]).
+* Adds tooltips to Discover button icons ({kibana-pull}192963[#192963]).
+* Excludes inactive integration data stream suggestions ({kibana-pull}192953[#192953]).
+* Fixes new variables being suggested in incorrect places ({kibana-pull}192405[#192405]).
+* Only log requests in the Inspector when they completed ({kibana-pull}191232[#191232]).
+* Fixes an issue where the autocomplete suggestions could cause duplicate entries in ES|QL queries ({kibana-pull}190465[#190465]).
```

Comment:
This one and the one after are not Discover bug fixes but editor fixes. The editor exists in many places, not only in Discover. Can we possibly have an ES|QL editor section?

Reply:
I was considering splitting ES|QL from Discover because of the number of entries. Do you think we should move just the "editor" ones or everything ES|QL-related?

Reply:
Hmmm, I think just the editor ones is better as then we will need to specify that this is for Discover, the other is for Visualizations etc. 

So my recommendation is:
- gather the editor's ones in a section. These apply for all apps using the editor
- keep in Discover the ES|QL things that concern only Discover
- keep in Visualizations the ES|QL things that concern only Vis

Reply:
Yeah, I think it's clearer now after applying your suggestion. Let me know what you think, and thanks for the detailed review!!